CRM/Upgrade/Incremental/php/FiveFortyNine.php

```diff
@@ -75,8 +82,8 @@ public function upgrade_5_49_beta1($rev): void {
    *
    * @param string $rev
    */
-  public function upgrade_5_49_2($rev) {
-    if (version_compare(CRM_Core_BAO_Domain::version(), '5.49', '>=') {
+  public function upgrade_5_49_0($rev) {
+    if (version_compare(CRM_Core_BAO_Domain::version(), '5.49.beta1', '>=')) {
```

Comment:
Well... that was worth a try... In my `r-run` over each of the upgrade paths, it never seems to return TRUE, so it always skipped the step.

I think I found a hidden feature which what this is trying to do... but still putting it through the `r-run`...

Reply:
Awesome, didn't knw that